Add warp.dev to default terminals

bennyweise opened the pull request:
Warp.dev recently released their terminal in Linux as a beta. This change just adds it to the default set of terminals.

Not sure if this is useful as a PR, feel free to close it out if not.

RedBearAK commented:
@bennyweise 

What is the purpose of having the `warp.dev` in the commented-out version? Is this one of those apps that has a different app class in X11/Xorg versus Wayland? I've seen several like that. 

That old commented list should actually be removed at some point. But if this app has multiple associated app class strings depending on the environment, they should be combined like this:

```py
{clas:"^dev.warp.Warp$|^warp.dev$" },
```

I would like to accept the PR so you can be listed as a contributor, but I just need to fully understand it first.

bennyweise commented:
> @bennyweise
> 
> What is the purpose of having the `warp.dev` in the commented-out version? Is this one of those apps that has a different app class in X11/Xorg versus Wayland? I've seen several like that.
> 
> That old commented list should actually be removed at some point. But if this app has multiple associated app class strings depending on the environment, they should be combined like this:
> 
> ```python
> {clas:"^dev.warp.Warp$|^warp.dev$" },
> ```
> 
> I would like to accept the PR so you can be listed as a contributor, but I just need to fully understand it first.

Thanks for the response - I'm not sure what I was thinking when I added it to the commented out list. I've cleaned that up and put it in alphabetical order now. I've only tried it out on Wayland so I'm not sure if there is anything different in X11.

RedBearAK commented:
@bennyweise 

> not sure what I was thinking when I added it to the commented out list

Well, you know the old saying, "Those who do not completely understand the structure of the Toshy config file are doomed to make minor mistakes on their PR." 

Looks good to me, and I appreciate you taking the time to submit the new terminal app class. It may seem like a very simple addition, but updating app classes is actually one of the most obvious ways for the end user to see benefits from an update of the default config, and changes like this will get intelligently merged into the user's config upon reinstalling Toshy from a new zip. 

If you were to submit something more complicated I would probably ask you to rebase it onto the `dev_beta` branch, since that is where I typically put new commits so I can test them in VMs before merging with `main`. I should probably add a section to the README about that. But in cases like this it's no problem to push it straight to `main`.
